packages/utils/models.js

```diff
@@ -230,7 +319,7 @@ export const ONLY_TOUCH = [MODELS.D199];
  * @returns {boolean}
  */
 export function hasOnlyTouch() {
-  return !VerifyMethodOnSystemWrapper('hasKeyboard') || _hasModelAtList(ONLY_TOUCH);
+  return VerifyMethodOnSystemWrapper('hasOnlyTouch') || !_hasModelAtList(HAS_KEYBOARD);
```

Comment:
Cara, o back não vai implementar o `hasOnlyTouch`, eles vão implementar só alguns, eu só sei que é o `hasSmallScreen`, e talvez um outro... tem que ver na thread la... Nesse caso aqui, eles falaram prá usar a negação do `hasKeyboard` mesmo. Então não adianta tentar chamar `VerifyMethodOnSystemWrapper('hasOnlyTouch')` que sempre vai dar errado.

```suggestion
  return !hasKeyboard();
```